ota/common.js

```diff
@@ -143,8 +143,11 @@ async function isUpdateAvailable(device, logger, isNewImageAvailable, requestPay
     }
 
     const available = await isNewImageAvailable(requestPayload, logger, device);
-    logger.debug(`Updata available for '${device.ieeeAddr}': ${available ? 'YES' : 'NO'}`);
-    return available;
+    logger.debug(`Update available for '${device.ieeeAddr}': ${available < 0 ? 'YES' : 'NO'}`);
+    if (available > 0) {
+        logger.debug(`Firmware on '${device.ieeeAddr}' is newer than latest firmware online.`);
```

Comment:
This probably deserves to be a warning rather than debug level message?

Reply:
done, thanks